recipe/meta.yaml

```diff
@@ -134,7 +131,7 @@ outputs:
       run:
         - python
         - {{ pin_subpackage('gymnasium', exact=True) }}
-        - mujoco-python >=2.2.0,<2.4
+        - mujoco-python >=2.3.2,<2.4
```

Comment:
I think there is a difference between mujoco and mujoco-python:
https://github.com/Farama-Foundation/Gymnasium/blob/cab0fbb13d8f3c29c098939144fcbb183bede7f2/pyproject.toml#L44 
https://github.com/Farama-Foundation/Gymnasium/blob/cab0fbb13d8f3c29c098939144fcbb183bede7f2/pyproject.toml#L46

mujoco-python has a lower constraint and is upper bound to 2.2.

Reply:
This one is confusing. There is a project called [mujoco-py](https://github.com/openai/mujoco-py) and there is a project called [mujoco](https://github.com/deepmind/mujoco). There is also https://anaconda.org/conda-forge/mujoco-python and https://anaconda.org/conda-forge/mujoco which are both from deepmind.

On PyPI, `mujoco` corresponds to the DeepMind project and `mujoco-py` corresponds to the OpenAI project.

I don't think the situation could be more confusing than that...

So unless I'm mistaken, I think `>=2.3.2,<2.4` is correct because in conda-forge, `mujoco-python` corresponds to https://pypi.org/project/mujoco/ in PyPI.

Or am I confused even more after writing all this? This is entirely possible.

Reply:
The `gymnasium-mujoco` subpackage definitely requires **mujoco-python** made by DeepMind and we have it https://github.com/AnacondaRecipes/mujoco-feedstock/blob/main/recipe/meta.yaml. 
The `gymnasium-mujoco_py` subpackage definitely requires
`mujoco-py` which isn't maintained, see https://github.com/Farama-Foundation/Gymnasium/pull/112#discussion_r1036369659. But yes, `mujoco-py` in `pyproject.toml` is `mujoco-py` made by OpenAI https://github.com/openai/mujoco-py#install-and-use-mujoco-py

Reply:
Could you enable `gymnasium-mujoco` subpackage because we have **mujoco-python 2.3.2**?